testData/qss/settings.json

```diff
@@ -177,6 +177,65 @@
             }
         },
         "learnMoreLink": "https://morphic.world/help/qsshelp#contrast"
+    }, {
+        "id": "mouse",
+        "path": "mouse",
+        "schema": {
+            "type": "mouse"
+        },
+        "settings": {
+            "mouseSpeed": {
+                "id": "mouseSpeed",
+                "path": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.PointerSpeed",
+                "schema": {
+                    "type": "number",
+                    "default": 10,
+                    "min": 1,
+                    "max": 20,
+                    "divisibleBy": 2
+                },
+                "value": 10,
```

Comment:
What is the separate meaning to the "value" entry here, in addition to the default value of the schema? I'm trying to remember why we didn't make use of the schema default value for the existing ordinary settings.

Reply:
Yeah... this is legacy thing, its used in all of the current settings. We will try to remove it from settings.json (I mean the generic "value" there) and use schema.default instead. I will keep you posted on this.

Reply:
Please write this up as a separate JIRA we can track after this pull goes in

Reply:
Bump comment!

Reply:
https://issues.gpii.net/browse/GPII-4252